airflow/providers/amazon/aws/transfers/imap_attachment_to_s3.py

```diff
@@ -51,24 +57,31 @@ def __init__(
         self,
         *,
         imap_attachment_name: str,
+        s3_bucket: str,
         s3_key: str,
         imap_check_regex: bool = False,
         imap_mail_folder: str = 'INBOX',
         imap_mail_filter: str = 'All',
         s3_overwrite: bool = False,
         imap_conn_id: str = 'imap_default',
-        s3_conn_id: str = 'aws_default',
+        s3_conn_id: Optional[str] = None,
```

Comment:
What is the reason to change the default?

Reply:
To raise the warning in case it is used. Check line 72